src/ansys/dpf/core/plotter.py

```diff
@@ -355,11 +355,6 @@ def show_figure(self, **kwargs):
             self._plotter.camera_position = cpos
 
         # Show depending on return_cpos option
-        plane = kwargs.pop("plane", None)
```

Comment:
@PProfizi it looks good to me, thank you ! It would be great if we could also update cpos to have the image zoomed, and an explanation of what has been done to get the corresponding cpos.